.travis.yml

```diff
@@ -1,7 +1,19 @@
+env:
+  global:
+    - NODE_ENV=test
+    - secret = authorshaven
+    - DATABASE_TEST=postgres://postgres:admin5423/users_test
+    - APP_URL_BACKEND = http://localhost:3000
 language: node_js
-node_js: 
+node_js:
   - "stable"
 services:
   - postgresql
+before_script:
+  - psql -c 'CREATE DATABASE users_test;' -U postgres
+  - npm i sequelize-cli -g
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
 after_success:
-  - npm test
```

Comment:
The presence of this line theoretically is incorrect in all manner, after_success means the command travis will run after running our script. Please we should only have one command here